client/post-editor/editor-revisions/dialog.jsx

```diff
@@ -79,7 +79,11 @@ class PostRevisionsDialog extends PureComponent {
 	dialogButtons = () => {
 		const { postId, revision, siteId, translate } = this.props;
 		return [
-			{ action: 'cancel', compact: true, label: translate( 'Cancel' ) },
+			{
+				action: 'cancel',
+				compact: true,
+				label: translate( 'Cancel', { context: 'modal button' } ),
```

Comment:
That was is not necessary - "Load" was the 2nd string I pointed out.

Reply:
Oops good spot! Thanks!

Reply:
Thanks @yoavf I've removed this change and now added the context to 'Load' :)